requirements.txt

```diff
@@ -1,9 +1,7 @@
 joblib==1.4.2
-numpy==1.24.4; python_version <= '3.8'
-numpy==1.26.4; python_version >= '3.9'
+numpy==2.0.2; python_version < '3.10'
+numpy==2.1.1; python_version >= '3.10'
 # Used for the training script:
-pandas==2.0.3; python_version < '3.8'
-pandas==2.2.2; python_version >= '3.9'
-scikit-learn==1.3.2; python_version <= '3.8'
```

Comment:
I think we should deprecate Python3.8 as we do not offer any value to people installing it, they get the same version and essentially +1 download. For this we need to update Readme.

Should we do it now or as a follow up?